release_notes.md

```diff
@@ -15,4 +15,8 @@ This file tracks release notes for the loop client.
 
 #### Breaking Changes
 
+* Macaroon authentication has been enabled for the `loopd` gRPC and REST
```

Comment:
Perhaps we want to mention that it will pick up by default on mainnet, but testnet needs the flag?

Reply:
Okay, I added the comment about the network. The rest (what directory things are located and so on) is described in the linked documentation so I think that should be fine.

Reply:
I think this was already the case for 0.9.0.